cmd/tailscale/cli/jsonoutput/network-lock-status.go

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !ts_omit_tailnetlock
+
+package jsonoutput
+
+import (
+	"encoding/base64"
+	jsonv1 "encoding/json"
+	"fmt"
+	"io"
+
+	"tailscale.com/ipn/ipnstate"
+	"tailscale.com/tka"
+)
+
+// PrintNetworkLockStatusJSONV1 prints the current Tailnet Lock status
+// as a JSON object to the CLI, in a stable "v1" format.
+func PrintNetworkLockStatusJSONV1(out io.Writer, status *ipnstate.NetworkLockStatus) error {
+	responseEnvelope := ResponseEnvelope{
+		SchemaVersion: "1",
+	}
+
+	var result any
+	if status.Enabled {
+		result = struct {
+			ResponseEnvelope
+			tailnetLockStatusV1
+		}{
+			ResponseEnvelope:    responseEnvelope,
+			tailnetLockStatusV1: toTailnetLockStatusV1(status),
+		}
+	} else {
+		result = struct {
+			ResponseEnvelope
+			Enabled bool
```

Comment:
I think it might be helpful to print `PublicKey` and `NodeKey` even if Tailnet Lock is not enabled. Any thoughts about always doing `toTailnetLockStatusV1(status)`?